src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,47 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] arr = new int[matrix.length];
+        int idxArr = 0;
```

Comment:
Use `i` instead of this, so we don't create redundant variables

Reply:
Done